.circleci/config.yml

```diff
@@ -429,12 +429,6 @@ jobs:
               --exclude mc-sgx-ias-types \
               --exclude mc-sgx-report-cache-untrusted \
               --exclude mc-sgx-urts-sys
-      - run:
```

Comment:
Removing testnet-client since we don't use this anymore, and this change would have required updating it.